tests/components/flexmeasures/test_services.py

```diff
@@ -52,11 +48,11 @@ async def test_trigger_and_get_schedule(
             blocking=True,
         )
         tzinfo = dt_util.get_time_zone(hass.config.time_zone)
-        mocked_FlexmeasuresClient.assert_called_with(
+        mocked_FlexmeasuresClient.assert_awaited_with(
             sensor_id=1,
             start=time_ceil(datetime.now(tz=tzinfo), pd.Timedelta(RESOLUTION)),
```

Comment:
Is this the only place where we use Pandas? If so, perhaps we could avoid a dependency? Here, possibly use `datetime.timedelta(RESOLUTION)`.